Stratis.Bitcoin.Features.Miner/PosMinting.cs

```diff
@@ -495,20 +495,28 @@ private bool SignBlock(List<StakeTx> stakeTxes, Block block, ChainedBlock pindex
             // it sorts the coins from heighest weight.
             setCoins = setCoins.OrderByDescending(o => o.TxOut.Value).ToList();
 
-            this.logger.LogTrace("Trying to find staking solution among {0} transactions.", setCoins.Count);
+            long minimalAllowedTime = pindexBest.Header.Time + 1;
```

Comment:
Very nice this can potentially even go in the top of this method (it can even go in the call before setCoins is created)

Reply:
Yes this is done in followup PR